File: B-example-solutions/obj-c/01/Assignment-01-04/Assignment-01-04/ViewController.m
//
//  ViewController.m
//  Assignment-01-04
//
//  Created by Philip Dow on 8/28/15.
//  Copyright (c) 2015 Phil Dow. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    // You could set this property in interface builder
    self.imageView.contentMode = UIViewContentModeScaleAspectFill;
    
    // Look in Images.xcassets for the bokeh image
    self.imageView.image = [UIImage imageNamed:@"bokeh"];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: B-example-solutions/swift/01/Assignment-01-04/Assignment-01-04/ViewController.swift
//
//  ViewController.swift
//  Assignment-01-04
//
//  Created by Philip Dow on 8/28/15.
//  Copyright (c) 2015 Phil Dow. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    @IBOutlet weak var imageView: UIImageView!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        // You could set this property in interface builder
        imageView.contentMode = .ScaleAspectFill
        
        // Look in Images.xcassets for the bokeh image
        imageView.image = UIImage(named: "bokeh")
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}

